### DISCUSSION TITLE
[5.0] Replace the accessibility icon: universal-access instead of wheelchair

### DISCUSSION THREAD

**Discussion by ChristineWk** [joomla/joomla-cms#46994] [2. Cold Features]
### Steps to reproduce the issue
How can the icon ("wheelchair user") used by default in the frontend be changed to another accessibility icon without a wheelchair?

Or maybe as an alternative for choosing.

### Expected result
Replace the accessibility icon: universal-access instead of wheelchair

### Actual result
![screen shot 2023-07-19 at 21 20 06](https://issues.joomla.org/uploads/1/aaf3c8e0fd88e4a0f102274e87066cbb.jpg)

### Additional comments
There is a PR #34713, but this is for the backend.

**Comment by chmst** [comment]
https://github.com/joomla/joomla-cms/pull/34713 had another context.

Relevant: https://github.com/joomla/joomla-cms/pull/39615

**Comment by sandewt** [comment]
The ♿ symbol is a unicode symbol and is embedded in the \media\vendor\accessibility\js\accessibility.js file

I could not find a suitable replacement symbol, see:
- [https://www.getemojis.net/emoji/wheelchair-symbol/](https://www.getemojis.net/emoji/wheelchair-symbol/)
- [https://www.getemojis.net/html/#Emoji-Liste](https://www.getemojis.net/html/#Emoji-Liste)

**Comment by ChristineWk** [comment]
@chmst Thank you for your info.

@sandewt Thank you for your interest. Interesting links. Also looked at (before your message): yooa11y.css and yoaa11.min.css - but difficult.

Btw: In Forum.de the alternative question came up, whether or how to change the fontawesome icon, which is gray, to a different background color? Couldn't find it yet, possibly because of CSS.
Here: ...\media\vendor\accessibility\js\accessibility.js file I haven't looked yet. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41191">issues.joomla.org/tracker/joomla-cms/41191</a>.</sub>

**Comment by sandewt** [comment]
> Btw: In Forum.de the alternative question came up, whether or how to change the fontawesome icon, which is gray, to a different background color? Couldn't find it yet, possibly because of CSS.

See: [https://fontawesome.com/v5/icons/universal-access?f=classic&s=solid](https://fontawesome.com/v5/icons/universal-access?f=classic&s=solid)

**Comment by ChristineWk** [comment]
@sandewt Thank you for your link to fontawesome.
I tried:

`i.fa-solid.universal-access {
content:"\f29a" 
}`
of course no success :)

Saw also:

> To use the regular style of “universal-access“, you'll need a subscription to a Pro-level plan or a perpetual Pro license that includes the specific version of Font Awesome in which this icon (or style) was released.

**Comment by sandewt** [comment]
> of course no success :)

Indeed, it only works with inline script / styling.

**Comment by ChristineWk** [comment]
> > of course no success :)
> 
> Indeed, it only works with inline script / styling.

"of course" :-) didn't mention before that I also entered the following html code:

`<i class="fa-solid fa-universal-access"></i>`

However, it was only temporary (F12)
either it is wrong or it has been placed in the wrong place.

**Comment by sandewt** [comment]
Maybe I misunderstand you. But this inline code shows a red background of the icon.

`<i class="fas fa-universal-access" style="color: #ff0000;"></i>`

**Comment by ChristineWk** [comment]
? My code starts do not have: ..."fas" .... Tried with "fa-solid" etc.
Hv to check again. Thank you.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41191">issues.joomla.org/tracker/joomla-cms/41191</a>.</sub>
